.bazelrc

```diff
@@ -12,38 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+###############################################################################
+# Common flags that apply to all configurations.
+# Use sparingly because they cannot be overriden.
+###############################################################################
+# Prevent invalid caching if input files are modified during a build.
+build --experimental_guard_against_concurrent_changes
+# Disable visibility checks (works around some private deps in TensorFlow that
+# are being unbundled soon anyway).
+build --nocheck_visibility
+build --compilation_mode=opt
+# Used in TensorFlow, so we have to enable it here as well.
+common --experimental_repo_remote_exec
+# Actually printing output on errors is... a useful default
+test --test_output=errors
```

Comment:
I usually run with this, but I'm not sure we should be changing the defaults here. This is a development workflow preference thing and seems orthogonal to this change. If I'm running 1000 tests I don't necessary want to see the errors inline. They'll usually be off the screen anyway after the final test results.

Reply:
I feel moderately strongly about this. I've seen enough engineers be shocked and awed to find out that such a thing exists to treat it as a poor default. If people would like it the other way, they can disable it :)